recipe/meta.yaml

```
@@ -10,33 +10,43 @@ source:
   sha256: eebca7f587b1761ce43a98ffd5327f2b600b9aa5bb0a77b947687f503ad43bc0
 
 build:
-  noarch: python
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
 
 requirements:
   host:
-    - python >=3.8
+    - python
     - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - cryptography
+    - pycryptodome
 
 test:
+  source_files: 
+    - tests
   imports:
     - joserfc
   commands:
     - pip check
+    - pytest -vv
   requires:
     - pip
+    - pytest
 
 about:
   home: https://jose.authlib.org/en/dev/
-  summary: The ultimate Python library for JOSE RFCs, including JWS, JWE, JWK, JWA, JWT
+  summary: Implementations of JOSE RFCs in Python
+  description: |
+    joserfc is a Python library that provides a comprehensive implementation of several essential JSON Object Signing and Encryption (JOSE) standards.
   dev_url: https://github.com/authlib/joserfc
+  doc_url: https://jose.authlib.org/en/dev/
```

```suggestion
  doc_url: https://jose.authlib.org
```
Minor, but this is friendlier to non-english users.